content/en/monitors/guide/troubleshooting-monitor-alerts.md

```diff
@@ -0,0 +1,99 @@
+---
+
+## Overview
+
+This guide provides an overview of some foundational concepts that can help you determine if your monitor's alerting behavior is valid. If you suspect that your monitor's evaluations are not accurately reflecting the underlying data, refer to the sections below as you inspect your monitor.
+
+### Monitor state and monitor status
+
+While monitor *evaluations* are stateless, meaning that the result of a given evaluation does not depend on the results of previous evaluations, monitors themselves are stateful, and their state is updated based on the evaluation results of their queries and configurations. A monitor evaluation with a given status won't necessarily cause the monitor's state to change to the same status. See below for some potential causes:
+
+1. **Metrics are too sparse within a metric monitor's evaluation window**
+
+If metrics are absent from a monitor's evaluation window, and the monitor is not configured to anticipate [no-data conditions][1], the evaluation may be `skipped`. In such a case, the monitor state is not updated, so a monitor previously in `OK` state remains `OK`, and likewise with a monitor in `Alert` state. Use the [history][2] graph on the monitor status page and select the group and time frame of interest. If data is sparsely populated, see [monitor arithmetic and sparse metrics][3] for more information.
+
+2. **Monitor state updates due to external conditions**
+
+The state of a monitor may also sometimes update in the absence of a monitor evaluation, for example, due to [auto-resolve][4]. 
+
+### Alert conditions
+
+Unexpected monitor behavior can sometimes be the result of misconfigured [alert conditions][5], which vary by [monitor type][6]. If your monitor query uses the `as_count()` function, check the [`as_count()` in Monitor Evaluations][7] guide. 
+
+If using recovery thresholds, check the conditions listed in the [recovery thresholds guide][8] to see if the behavior is expected.
+
+### Monitor status and groups
+
+For both monitor evaluations and state, status is tracked by group. 
+
+For a multi-alert monitor, a group is a set of tags with one value for each grouping key (for example, `env:dev, host:myhost` for a monitor grouped by `env` and `host`). For a simple alert, there is only one group (`*`), representing everything within the monitor's scope. 
+
+By default, Datadog keeps monitor groups available in the UI for 24 hours, or 48 hours for host monitors, unless the query is changed. See [Monitor settings changes not taking effect][9] for more information.
+
+If you anticipate new monitor groups to be created within the scope of your multi-alert monitors, you may want to configure a delay for the evaluation of these new groups. This can help you to avoid alerts from expected behavior of new groups, such as high resource usage associated with the creation of a new container. Read [new group delay][10] for more information.
```

Comment:
```suggestion
If you anticipate creating new monitor groups within the scope of your multi-alert monitors, you may want to configure a delay for the evaluation of these new groups. This can help you avoid alerts from the expected behavior of new groups, such as high resource usage associated with the creation of a new container. Read [new group delay][10] for more information.
```